casper/src/test/scala/coop/rchain/casper/addblock/MultiParentCasperAddBlockSpec.scala

```diff
@@ -483,7 +483,8 @@ class MultiParentCasperAddBlockSpec extends FlatSpec with Matchers with Inspecto
   it should "succeed at slashing" in effectTest {
     TestNode.networkEff(genesis, networkSize = 3).use { nodes =>
       for {
-        deployData   <- ConstructDeploy.basicDeployData[Effect](0)
+        deployData <- ConstructDeploy
+                       .basicDeployData[Effect](0, shardId = genesis.genesisBlock.shardId)
```

Comment:
I'm not sure - why this change is required? shard validation removed so this should not make any difference?

Reply:
Without that test will fail with invalid deploy shard ID error.